examples/multitask-monorepo/s3/s3.tf

```diff
@@ -0,0 +1,28 @@
+module "s3_logs" {
```

Comment:
Let's remove all items that are not nessesary to show multistack example. Make sure to rebase too.